content/internships/history/contents.lr

```diff
@@ -6,10 +6,16 @@ title: Open Source Internships: History
 ---
 body:
 
+## Google Season of Docs
+
+CC has participated in Google Season of Docs for the first time in 2020.
+
 ## Google Summer of Code
 
 CC has participated in Google Summer of Code eight times before 2020. See below for more information about our projects and participation for each year.
 
+* [Google Summer of Code 2020](https://summerofcode.withgoogle.com/organizations/5742311717208064/)
+    * [Open Source Blog posts](/blog/categories/gsoc-2020/)
```

Comment:
please also link to https://creativecommons.org/2020/05/11/welcome-interns-google-summer-of-code-and-outreachy/

Reply:
Resolved in f00524f.